modules/ossm-config-smcp-kiali.adoc

```diff
@@ -0,0 +1,204 @@
+|string
+|`kiali`
+
+|kiali:
+  enabled:
+|This parameter enables/disables Kiali. Kiali is enabled by default.
+|`true`/`false`
+|`true`
+
+|kiali:
+  install:
+|Install a Kiali resource if the named Kiali resource is not present. The `install` section is ignored if `addons.kiali.enabled` is set to `false`.
+|
+|
+
+|kiali:
+  install:
+    dashboard:
+|Configuration parameters for the dashboards shipped with Kiali.
+|
+|
+
+|kiali:
+  install:
+    dashboard:
+      viewOnly:
+|This parameter enables/disables view-only mode for the Kiali console. When view-only mode is enabled, users cannot use the Kiali console to make changes to the {SMProductShortname}.
+|`true`/`false`
+|`false`
+
+|kiali:
+  install:
+    dashboard:
+      enableGrafana:
+|Grafana endpoint configured based on `spec.addons.grafana` configuration.
+|`true`/`false`
+|`true`
+
+|kiali:
+  install:
+    dashboard:
+      enablePrometheus:
+|Prometheus endpoint configured based on `spec.addons.prometheus` configuration.
+|`true`/`false`
+|`true`
+
+|kiali:
+  install:
+    dashboard:
+      enableTracing:
+|Tracing endpoint configured based on Jaeger custom resource configuration.
+|`true`/`false`
+|`true`
+
+|kiali:
+  install:
+    service:
+|Configuration parameters for the Kubernetes service associated with the Kiali installation.
+|
+|
+
+|kiali:
+  install:
+    service:
+      metadata:
+|Use to specify additional metadata to apply to resources.
+|N/A
+|N/A
+
+|kiali:
+  install:
+    service:
+      metadata:
+        annotations:
+|Use to specify additional annotations to be applied to the component's service.
+|string
+|N/A
+
+|kiali:
+  install:
+    service:
+      metadata:
+        labels:
+|Use to specify additional labels to be applied to the component's service.
+|string
+|N/A
+
+|kiali:
+  install:
+    service:
+      ingress:
+|Used to specify details for accessing the component’s service through an OpenShift Route.
+|N/A
+|N/A
+
+|kiali:
+  install:
+    service:
+      ingress:
+        metadata:
+          annotations:
+|Use to specify additional annotations to be applied to the component's service ingress.
+|string
+|N/A
+
+|kiali:
+  install:
+    service:
+      ingress:
+        metadata:
+          labels:
+|Use to specify additional labels to be applied to the component's service ingress.
+|string
+|N/A
+
+|kiali:
+  install:
+    service:
+      ingress:
+        enabled:
+|Use to customize an OpenShift Route for the service associated with a component.
+|`true`/`false`
+|`true`
+
+|kiali:
+  install:
+    service:
+      ingress:
+        contextPath:
+|Use to specify the context path to the service.
+|string
+|N/A
+
+|install:
+  service:
+    ingress:
+      hosts:
+|Use to specify a single host name per OpenShift route. An empty host name implies a default host name for the Route.
```

Comment:
```suggestion
|Use to specify a single hostname per OpenShift route. An empty hostname implies a default host name for the Route.
```
Multiple instances.
IBM SG: hostname noun
Write as shown unless updating existing content that uses "host name".